obspy/taup/slowness_model.py

```diff
@@ -634,7 +637,7 @@ def depthInFluid(self, depth):
             otherwise.
         :rtype: :class:`~numpy.ndarray` (dtype = :class:`bool`)
         """
-        ret = np.zeros(shape=depth.shape, dtype=np.bool_)
+        ret = False
```

Comment:
When going through the insides for serialization I think I found this always is a single valued array.. but yeah, I did not really make sure, so I'll revert this change.